ts/screens/modal/components/IdentificationNumberPad.tsx

```diff
@@ -82,26 +82,19 @@ export const IdentificationNumberPad = (
           onValidate={onPinValidated}
         />
       </View>
-      <VSpacer size={48} />
-      <View>
-        <NumberPad
-          value={value}
-          deleteAccessibilityLabel={I18n.t("global.buttons.delete")}
-          onValueChange={onValueChange}
-          variant={numberPadVariant}
-          {...biometricsConfig}
-        />
-      </View>
       {isDevEnv && (
         <View
           accessible={false}
           importantForAccessibility="no-hide-descendants"
           accessibilityElementsHidden
-          style={IOStyles.alignCenter}
+          style={{
+            alignSelf: "center",
+            position: "absolute",
+            bottom: -56
+          }}
```

Comment:
We need to find an absolute position that it doesn't hide the "dev button" in small devices. For example in my nexus 5 I'm not able to tap on it, because it's totally hidden below the Android navigation bar. What do you think? Where can we place it?

<img width="250" src="https://github.com/pagopa/io-app/assets/16268789/86da631b-47b9-43b7-8c47-34e01a19f6dc" />

Reply:
Can we move it to the bottom right or left between `NumberPad` and `ButtonLink` below? I know it's not an aesthetically pleasing position, but we need to make it intentionally ugly so the developer understands it's a development button and nothing else. What do you think?

Reply:
Okay, let's try it out!

Reply:
Now LGTM 

<img width="250" src="https://github.com/pagopa/io-app/assets/16268789/39a10bd4-5557-47e2-82b7-f159dc04dfd0" />